lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -171,7 +173,8 @@ def redcap_det_uw_reopening(*, db: DatabaseSession, cache: TTLCache, det: dict,
             elif is_complete('test_order_survey', redcap_record_instance):
                 collection_method = CollectionMethod.SWAB_AND_SEND
         else:
-            LOG.warning(f"The record instance has an unexpected event name: {redcap_record_instance.event_name}")
+            LOG.warning(f"The record instance has an unexpected event name: '{redcap_record_instance.event_name}' "
```

Comment:
FWIW, using `{redcap_record_instance.event_name!r}` will produce a quoted string that's always unambiguous regardless of contents, unlike surrounding the interpolation with literal quotes.

Reply:
Oh cool. I saw the "!r" in something you wrote this week and didn't dig into what it does ... 'r' being the internal representation ... repr.